packages/labs/task/src/test/task_test.ts

```diff
@@ -777,24 +808,24 @@ suite('Task', () => {
     assert.equal(warnMessages.length, 0);
   });
 
-  test('performTask waits on the task', async () => {
-    const el = getTestElement({
-      args: () => [el.a],
-    });
-    await renderElement(el);
-    let taskComplete = false;
-    (async () => {
-      el.a = 'z';
-      // @ts-expect-error: We're testing the behavior of a protected method
-      await el.task.performTask();
-      taskComplete = true;
-    })();
-    await tasksUpdateComplete();
-    assert.isFalse(taskComplete);
-    el.resolveTask();
-    await tasksUpdateComplete();
-    assert.isTrue(taskComplete);
-  });
+  // test('performTask waits on the task', async () => {
```

Comment:
Is this intended?

Reply:
I'll remove. We should have coverage that various ways to run the task already wait for the task to complete.